airflow/jobs.py

```diff
@@ -786,6 +788,7 @@ def create_dag_run(self, dag, session=None):
             )
             # return if already reached maximum active runs and no timeout setting
             if len(active_runs) >= dag.max_active_runs and not dag.dagrun_timeout:
+                self.logger.info("Dag reached maximum active runs (no timeout)")
```

Comment:
@saguziel I did not put the dag_id in any of the logs to remain in line with the other logs and also because they are already split by their dag name.

Do you still want me to do this? If so, would you like all logs to include the dag_id to remain consistent?